.github/config.yml

```diff
@@ -0,0 +1,36 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
+  
+  
+  Thank you and congrats 🎉 for opening your very first issue in this project.AnitaB.org is an inclusive community, commited to creating a safe and positive environment.🌸 Please adhere to our [Code of Conduct](https://github.com/anitab-org/open-source-programs-web/blob/develop/CODE_OF_CONDUCT.md).🙌
+  You may submit a PR if you like! If you want to report a bug🐞 please follow our [Issue Template](https://github.com/anitab-org/.github/tree/main/.github/ISSUE_TEMPLATE). Also make sure you include steps to reproduce it and be patient while we get back to you.😄
```

Comment:
Just a few suggestions @Amulya-coder  there is no requirement of an exclamation `You may submit a PR if you like!` in this line